airgun/views/ansible_role.py

```diff
@@ -21,10 +21,10 @@ class AnsibleRolesView(BaseLoggedInView, SearchableViewMixin):
     is present, without the search widget or table.
     """
 
-    title = Text("//h1")
+    title = Text("//h1[text()='Ansible Roles']")
```

Comment:
Can we avoid the exact string match as if anything changes in string in future, then it doesn't cause failure.

Reply:
Makes sense!